### PR TITLE
chore: update tests to use blueprint test framework

### PR DESCRIPTION
**Pull request by bharathkkb**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@bharathkkb
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/examples/cloudiot/README.md /tmp/tmp.lAcY5G3F2a/generate_docs/workspace/examples/cloudiot/README.md
8d7
< | name | The name for the Cloud IoT registry | `string` | n/a | yes |
10d8
< | region | The region for the IoT resources | `string` | n/a | yes |
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/examples/simple/README.md /tmp/tmp.lAcY5G3F2a/generate_docs/workspace/examples/simple/README.md
11,12d10
< | topic\_labels | A map of labels to assign to the Pub/Sub topic | `map(string)` | `{}` | no |
< | topic\_name | The name for the Pub/Sub topic | `string` | n/a | yes |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ``` 
- ⚠️check_headers
 All files need a license header. Please make sure all your files include the appropriate header. A helper tool available [here](https://github.com/google/addlicense). 
 ```bash 
Checking file headers
./test/integration/discover_test.go
./test/integration/simple/simple_test.go
./test/integration/cloudiot/cloudiot_test.go
 ``` 
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
examples/simple/main.tf
--- old/examples/simple/main.tf
+++ new/examples/simple/main.tf
@@ -19,9 +19,9 @@
 }
 
 module "pubsub" {
-  source       = "../../"
-  project_id   = var.project_id
-  topic        = "cft-tf-pubsub-topic"
+  source     = "../../"
+  project_id = var.project_id
+  topic      = "cft-tf-pubsub-topic"
   topic_labels = {
     foo_label = "foo_value"
     bar_label = "bar_value"
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

File: test/integration/cloudiot/cloudiot_test.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCloudiot(t *testing.T) {
```
Comment (morgante) [inline test/integration/cloudiot/cloudiot_test.go:12]: nit

```suggestion
func TestCloudIot(t *testing.T) {
```

**Review by morgante — APPROVED** [review]

**Comment by comment-bot-dev** [comment]
@bharathkkb
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/examples/cloudiot/README.md /tmp/tmp.Ks9aUhjUhS/generate_docs/workspace/examples/cloudiot/README.md
8d7
< | name | The name for the Cloud IoT registry | `string` | n/a | yes |
10d8
< | region | The region for the IoT resources | `string` | n/a | yes |
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/examples/simple/README.md /tmp/tmp.Ks9aUhjUhS/generate_docs/workspace/examples/simple/README.md
11,12d10
< | topic\_labels | A map of labels to assign to the Pub/Sub topic | `map(string)` | `{}` | no |
< | topic\_name | The name for the Pub/Sub topic | `string` | n/a | yes |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ``` 
- ⚠️check_headers
 All files need a license header. Please make sure all your files include the appropriate header. A helper tool available [here](https://github.com/google/addlicense). 
 ```bash 
Checking file headers
./test/integration/simple/simple_test.go
./test/integration/cloudiot/cloudiot_test.go
./test/integration/discover_test.go
 ``` 
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
examples/simple/main.tf
--- old/examples/simple/main.tf
+++ new/examples/simple/main.tf
@@ -19,9 +19,9 @@
 }
 
 module "pubsub" {
-  source       = "../../"
-  project_id   = var.project_id
-  topic        = "cft-tf-pubsub-topic"
+  source     = "../../"
+  project_id = var.project_id
+  topic      = "cft-tf-pubsub-topic"
   topic_labels = {
     foo_label = "foo_value"
     bar_label = "bar_value"
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@bharathkkb
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Pull request merged**